test/distributed/_tensor/test_dtensor_ops.py

```diff
@@ -713,4 +713,7 @@ def check_dtensor_func(self, test_func, opinfo, dry_run=False):
 
 
 if __name__ == "__main__":
-    run_tests()
+    # NB: CPU dtensor ops test frequently timeout https://github.com/pytorch/pytorch/issues/98816
+    # so running it only on CUDA
+    if torch.cuda.is_available():
```

Comment:
hmmm I don't think `test_dtensor_ops` runs in CUDA, so this essentially means it would only run in CUDA available machines with cpu tests https://github.com/pytorch/pytorch/blob/master/test/distributed/_tensor/test_dtensor_ops.py#L559

Reply:
which I think it might be fine for short term while we are investigating the timeouts. I just want to make sure there's at least something running in the CI so that we can capture errors when submitting PRs :)

Reply:
Yeah, the test would still be run on machine with CUDA, which curiously is never getting timed out.  I will update the PR title accordingly

Reply:
Got it thanks!